src/KoIdentity.Abstraction/Controllers/IUserController.cs

```diff
@@ -0,0 +1,175 @@
+        /// <param name="cancellationToken">
+        /// The <see cref="CancellationToken"/> is being used to propagate notifications that the operation should be
+        /// canceled.
+        /// </param>
+        /// <returns>
+        /// The <see cref="Task{TResult}"/> that represents the asynchronous operation, containing the
+        /// <see cref="string"/> that contains information if the removal succeeded.
+        /// </returns>
+        public Task<string> Remove(Guid userId, CancellationToken cancellationToken = default);
+        
+        /// <summary>
+        /// Gets a user with the specified <paramref name="userId"/>.
+        /// </summary>
+        /// <param name="userId">The unique user-id of the user who should be retrieved.</param>
+        /// <param name="cancellationToken">
+        /// The <see cref="CancellationToken"/> is being used to propagate notifications that the operation should be
+        /// canceled.
+        /// </param>
+        /// <returns>
+        /// The <see cref="Task{TResult}"/> that represents the asynchronous operation, containing the
+        /// <typeparamref name="TUserDto"/>, if any, who has the specified <paramref name="userId"/>.
+        /// </returns>
+        public Task<TUserDto> Get(Guid userId, CancellationToken cancellationToken = default);
+        
+        /// <summary>
+        /// Updates a user with the specified <typeparamref name="TUserDto"/>
+        /// </summary>
+        /// <param name="user">The data transfer object being used containing the user.</param>
+        /// <param name="cancellationToken">
+        /// The <see cref="CancellationToken"/> is being used to propagate notifications that the operation should be
+        /// canceled.
+        /// </param>
+        /// <returns>
+        /// The <see cref="Task{TResult}"/> that represents the asynchronous operation, containing the
+        /// <see cref="string"/> that contains information if the update succeeded.
+        /// </returns>
+        public Task<string> Update(TUserDto user, CancellationToken cancellationToken = default);
+        
+        /// <summary>
+        /// Get all roles for an user with the specified <paramref name="userId"/>.
+        /// </summary>
+        /// <param name="userId">The unique user-id of the user whose roles should be retrieved.</param>
+        /// <param name="cancellationToken">
+        /// The <see cref="CancellationToken"/> is being used to propagate notifications that the operation should be
+        /// canceled.
+        /// </param>
+        /// <returns>
+        /// The <see cref="Task{TResult}"/> that represents the asynchronous operation, containing the
+        /// <see cref="List{T}"/> of <typeparamref name="TRole"/>´s, if any, whose user has the specified
+        /// <paramref name="userId"/>.
+        /// </returns>
+        public Task<List<TRole>> GetRoles(Guid userId, CancellationToken cancellationToken = default);
```

Comment:
Please change to the following

```suggestion
        public Task<List<TRole>> GetRolesForUser(Guid userId, CancellationToken cancellationToken = default);
```